niworkflows/interfaces/cifti.py

```diff
@@ -272,7 +134,262 @@ class CiftiNameSource(SimpleInterface):
 
     def _run_interface(self, runtime):
         suffix = 'bold.dtseries'
-        if 'hcp' in self.inputs.variant:
+        if 'hcp' in self.inputs.variant.lower():
             suffix = 'space-hcp_bold.dtseries'
```

Comment:
shouldn't this be?
```suggestion
             suffix = 'space-fsLR_bold.dtseries'
```

Reply:
the CIFTI has `fsLR` surfaces, but `MNI152NLin6Asym` volumes - HCP outputs are in the same space, though not sure what the best label for this would be

Reply:
The definition of `fsLR` implicitly defines `MNI152NLin6Asym` as the volumetric template reference. Hence, saying `space-fsLR` univocally prescribes that surfaces are from the HCP atlas and the volumetric reference is FSL's MNI.

In other words, I can confirm this should be `space-fsLR`.